default.nix

```diff
@@ -473,6 +474,9 @@ let
         '';
       };
 
+      all-hies = import (fetchTarball "https://github.com/reactormonk/all-hies/tarball/master") { inherit pkgs; lib = pkgs.lib; };
```

Comment:
Why from your fork? Also, we use [`niv`](https://github.com/nmattia/niv) for managing external git pins, so use that.

Reply:
I updated to the newest hie-bios. I think all-hies does pinning itself (given it runs cabal2nix). I haven't fully grasped it yet.

Reply:
When I say "external pins", I mean things like external git repositories that are fixed to a particular revision. Which, I notice, this isn't (it's just tracking master)! That's another thing we don't do: everything is *always* pinned to a specific revision, otherwise you lose reproducibility.

Make sure you leave a comment about why we're deviating from upstream (want a newer version, in this case), and under what conditions we should switch back (they get up to date).

Reply:
Should be fixed by using niv.